Gems/LyShineExamples/Code/CMakeLists.txt

```diff
@@ -42,8 +42,12 @@ ly_add_target(
 
 # if enabled, LyShineExamples is used by all kinds of applications, however, the dependency to LmbrCentral is different
 # per application type
-ly_create_alias(NAME LyShineExamples.Builders NAMESPACE Gem TARGETS Gem::LyShineExamples Gem::UiBasics.Builders Gem::LmbrCentral.Editor)
-ly_create_alias(NAME LyShineExamples.Tools    NAMESPACE Gem TARGETS Gem::LyShineExamples Gem::LmbrCentral.Editor)
+if (PAL_TRAIT_BUILD_HOST_TOOLS)
+    # only if building tools for this platform, alias to tool targets
+    ly_create_alias(NAME LyShineExamples.Builders NAMESPACE Gem TARGETS Gem::LyShineExamples Gem::UiBasics.Builders Gem::LmbrCentral.Editor)
```

Comment:
With these updates, the `Builders` alias can be  outside of the `PAL_TRAIT_BUILD_HOST_TOOLS` check right?

Reply:
Not sure what you mean by this.   This particular change just reverts it to what it was before.